### PR TITLE
Lore updates

### PR DESCRIPTION
**Pull request by SpadesNeil**
This PR is entirely stuff to update in-game content to match lore on the wiki.

### CODE REVIEW

File: code/modules/busy_space/organizations.dm [hunk 1]
```diff
@@ -435,9 +435,11 @@
 	destination_names = list(
 						"Venus",
 						"Earth",
+						"Mercury",
 						"Luna",
 						"Mars",
-						"Titan"
+						"Titan",
+						"Europa"
```
Comment (Arokha) [inline code/modules/busy_space/organizations.dm]: vorestation edits, or, put this in another file and just override that list

File: code/modules/paperwork/faxmachine.dm [hunk 2]
```diff
@@ -1,5 +1,5 @@
 var/list/obj/machinery/photocopier/faxmachine/allfaxes = list()
-var/list/admin_departments = list("[using_map.boss_name]", "Virgo Prime Governmental Authority", "Supply")
+var/list/admin_departments = list("[using_map.boss_name]", "Virgo-Prime Governmental Authority", "Supply")
```
Comment (Arokha) [inline code/modules/paperwork/faxmachine.dm]: This appears to have been an unmarked vorestation edit, better add one while you're there

File: code/modules/paperwork/faxmachine.dm [hunk 3]
```diff
@@ -182,7 +182,7 @@ var/list/adminfaxes = list()	//cache for faxes that have been sent to admins
 	//message badmins that a fax has arrived
 	if (destination == using_map.boss_name)
 		message_admins(sender, "[uppertext(using_map.boss_short)] FAX", rcvdcopy, "CentComFaxReply", "#006100")
-	else if (destination == "Virgo Prime Governmental Authority")
+	else if (destination == "Virgo-Prime Governmental Authority")
```
Comment (Arokha) [inline code/modules/paperwork/faxmachine.dm]: Same

File: code/modules/busy_space/organizations.dm (hunk 1 above)
Comment (SpadesNeil) [reply code/modules/busy_space/organizations.dm]: I think I added this to the wrong file. I overrode it in busy_space_virgo, or so I thought.

**Comment by Arokha** [comment]
Once more, with feeling!

File: code/modules/busy_space/organizations.dm [hunk 4]
```diff
@@ -435,6 +435,7 @@
 	destination_names = list(
 						"Venus",
 						"Earth",
+						"Mercury",
```
Comment (Arokha) [inline code/modules/busy_space/organizations.dm]: //VOREStation Edit (or remove this edit since you copied the stuff in this file anyway)

Comment (SpadesNeil) [reply code/modules/busy_space/organizations.dm]: Ah crap not again.

File: code/modules/lore_codex/lore_data_vr/important_locations.dm [hunk 5]
```diff
@@ -28,7 +28,7 @@
 
 /datum/lore/codex/page/radiance_energy_chain/add_content()
 	name = "R.E.C. (Artificial Satellites)"
-	keywords += list("Radiance Energy Chain")
+	keywords += list("Radiance Energy Chain", "R.E.C.")
 	data = "A sparse government-owned chain of automated stations exists between [quick_link("Virgo 3")] and the star itself, known as the Radiance Energy Chain. The idea is based on \
 	an ancient design that was pioneered at Sol. The stations are heavily shielded from the stellar radiation, and feature massive \
 	arrays of photo-voltaic panels. Each station harvests energy from Vir using the solar panels, and sends it to other areas of \
```
Comment (SpadesNeil) [inline code/modules/lore_codex/lore_data_vr/important_locations.dm]: I have to fix this line.

**Pull request merged**